src/getting-started/index.md

```diff
@@ -1,10 +1,19 @@
 # What is Authup?
 
-Authup is an authentication & authorization system.
-It can be used as a headless system and can be provided with its own user interface.
-Otherwise, the supplied user interface can be used.
+Authup is an easy to use yet powerful authentication & authorization system. It's modular design allow you to only
```

Comment:
Maybe we should name some example components e.g. ui, api, ... which can deployed in a modular fashion ?